app/app.go

```diff
@@ -68,6 +69,8 @@ func init() {
 		geo.AppModuleBasic{},
 		upgrade.AppModuleBasic{},
 	)
+
+	sdk.PowerReduction = sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(2), nil))
```

Comment:
So... I dug quite a bit to the usage of this variable.

It seems that:

1. Setting it in the constructor is the best way to override AND
2. You do not actually need to re-calculate all the validators powers, as this is done every EndBlock automatically, so will be updated at the end of the first block after the upgrade.

```
// Apply and return accumulated updates to the bonded validator set. Also,
// * Updates the active valset as keyed by LastValidatorPowerKey.
// * Updates the total power as keyed by LastTotalPowerKey.
// * Updates validator status' according to updated powers.
// * Updates the fee pool bonded vs not-bonded tokens.
// * Updates relevant indices.
// It gets called once after genesis, another time maybe after genesis transactions,
// then once at every EndBlock.
//
// CONTRACT: Only validators with non-zero power or zero-power that were bonded
// at the previous block height or were removed from the validator set entirely
// are returned to Tendermint.
func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []abci.ValidatorUpdate) {
```

Reply:
Adding a short comment before this line would not hurt however.

Reply:
Okay, so this seems like a correct way to do this? It's in an `init` function.

Reply:
Anything before the first `EndBlock` call should work well.

`init` is good to ensure that the default sdk value is never read anywhere